virtool/indexes/api.py

```diff
@@ -31,89 +32,100 @@
 routes = Routes()
 
 
-@routes.get("/indexes")
-async def find(req):
-    """
-    Return a list of indexes.
+@routes.view("/indexes")
+class IndexesView(PydanticView):
+    async def get(self) -> Union[r200[GetIndexesResponse]]:
+        """
+        Find indexes.
 
-    """
-    db = req.app["db"]
+        Return a list of indexes.
 
-    ready = req.query.get("ready", False)
+        Status Codes:
+            200: Successful operation
 
-    if not ready:
-        data = await virtool.indexes.db.find(db, req.query)
-        return json_response(data)
+        """
+        db = self.request.app["db"]
 
-    pipeline = [
-        {"$match": {"ready": True}},
-        {"$sort": {"version": -1}},
-        {
-            "$group": {
-                "_id": "$reference.id",
-                "index": {"$first": "$_id"},
-                "version": {"$first": "$version"},
-            }
-        },
-    ]
+        ready = self.request.query.get("ready", False)
```

Comment:
Use `aiohttp-pydantic` support for query strings.

https://github.com/Maillol/aiohttp-pydantic#inject-query-string-parameters